base/util.jl

```diff
@@ -630,6 +633,7 @@ macro kwdef(expr)
     return quote
         $(esc(:($Base.@__doc__ $expr)))
         $kwdefs
+        $(esc(:(Base.kwdef_defaults)))(::Type{$(esc(S))}) = (;$(filter(p -> isexpr(p, :kw), parameters)...))
```

Comment:
```suggestion
        $(esc(:($Base.kwdef_defaults)))(::Type{$(esc(S))}) = (;$(filter(p -> isexpr(p, :kw), parameters)...))
```
As is done for `Base.@__doc__` a couple lines up. This ensures that it works even if `Base` isn't defined or refers to something else.